BetaX-Community/editor
# Added Code Formatter

**What does this PR do?**
This PR added the code formatter.

Fixes #19 , #12

**Type of change**
This PR fixed the code by formatting them.

**How should this be tested?**

push code.
Check it on GitHub Action.

**To enable the Auto-commit Action for creating a commit with formatted changes, we need to have the appropriate write permissions. By default, it is set to read-only. To verify this, navigate to Settings -> Actions -> General and ensure that the workflow has both read and write permissions enabled. Now, whenever there is a push (either through commit or direct push) or a pull request, the workflow will automatically execute.**

**Mandatory Tasks**
